content/snowcamp.fr.html

```diff
@@ -80,13 +80,9 @@ <h2>Les conférences</h2>
                 <li>L'unconference, le samedi, propose aux orateurs et aux participants de se retrouver sur les
                   pistes de ski de la région grenobloise.</li>
               </ul>
-              <p>Les trois premières éditions du SnowCamp ont réuni :</p>
-              <ul>
-                <li>2016 : ~200 participants et orateurs</li>
-                <li>2017 : ~300 participants et orateurs</li>
-                <li>2018 : ~450 participants et orateurs</li>
-                <li>2019 : ~450 participants et orateurs</li>
-              </ul>
+              <p>La première édition en 2016 réunissait ~200 participants et orateurs au complexe de recherche
+                Minatec et a grandi grâce à votre confiance pour pouvoir accueillir ~450 participants et ~70
+                orateurs en 2022 au World Trade Center de Grenoble.</p>
```

Comment:
```suggestion
              <p>La première édition en 2016 réunissait ~200 participants et orateurs dans les locaux de l'université de Grenoble et a grandi grâce à votre confiance pour pouvoir accueillir ~450 participants et ~70
                orateurs en 2022 au World Trade Center de Grenoble.</p>
```

Reply:
content fix - first edition took place on the university campus, not in Minatec